versions.yaml

```diff
@@ -3,7 +3,7 @@ SCCACHE_VER: 0.7.7
 # renovate: datasource=github-releases depName=cli/cli
 GH_CLI_VER: 2.51.0
 # renovate: datasource=github-releases depName=codecov/uploader
```

Comment:
We need to point renovate to the new source, which comes from PyPI. I don't know the proper syntax for this, but here is my best guess. Please verify this!
```suggestion
# renovate: datasource=pypi depName=codecov-cli
```

Reply:
Yes this seems correct ([src](https://docs.renovatebot.com/modules/datasource/pypi/))